### ISSUE TITLE
Disable spinners when not outputting to a TTY

### ISSUE THREAD

**Issue by vidyarani-dg** [ManrajGrover/halo#46]
I am using halo for the spinners and it works absolutely fine while running the management command from the command line, however I do not want the spinners while running my test suite. This is cluttering my tests and I want to disable it. I don't see an option to disable it. The only other option is to write the output to a file.

Is it possible to disable the spinners when not outputting to a TTY ? 

## Description
- Disable spinners when not outputting to a TTY

### System settings

- Operating System: Ubuntu 16.04.3 LTS
- Python version: 3.6.3
- Halo version: 0.0.10

**Comment by ManrajGrover** [comment]
@vidyarani-dg We do have an option for [enabling and disabling](https://github.com/ManrajGrover/halo/blob/master/halo/halo.py#L66) the [spinner](https://github.com/ManrajGrover/halo#enabled). I believe you can disable it in your test environment using this.

**Comment by vidyarani-dg** [comment]
I see , thanks! It's clearly in the initialization. Sorry, I missed it. 

Will close this issue.

**Comment by ManrajGrover** [comment]
@vidyarani-dg Did it work as per your requirement?

**Comment by vidyarani-dg** [comment]
@ManrajGrover : Apologies for the tardiness in getting back to you, I did try 

``
Halo(text='Loading files', enabled=False) 
``

It did disable the info messages, however the errors are still outputting to the terminal. 

Is this how it is supposed to work ?

**Comment by ManrajGrover** [comment]
@vidyarani-dg Which errors are being outputted? I would need more information in order to assist.

**Comment by vidyarani-dg** [comment]
@ManrajGrover : Sorry, the failure messages . For instance, this is how I am using the spinner. 

With enabled=False, I do not see any of the success / text messages on the terminal, however I do see the failure messages.

```
spinner = Halo(text='Loading tramada files', dots', open(os.devnull, "w"))
spinner.start()
spinner.text = 'Processing line {}'.format(stage_three.rownum+2)

for row in rows:
    try:
        load(row)
    except Exception:
        spinner.fail("Error loading row {}".format(message)) # -> This is outputted to the terminal. 
        spinner.start()

spinner.succeed('Load successful, time taken in seconds: {}'.format((end - start).seconds))
spinner.stop()
```

**Comment by ManrajGrover** [comment]
@vidyarani-dg Got it. Thanks for reporting this. I'll fix this and notify you with the release.

**Comment by vidyarani-dg** [comment]
Thank you.

**Comment by ManrajGrover** [comment]
@vidyarani-dg Made a release with fix in v0.0.11. Please upgrade and check if it works as expected.
